### DISCUSSION TITLE
How and where should we maintain the project roadmap?

### DISCUSSION THREAD

**Discussion by paulhowardarm** [parallaxsecond/parsec#459] [General]
The project needs an open and collaborative roadmap, where all contributors and stakeholders have access to an at-a-glance view of what projects are in flight, and what the upcoming deliverables are with approximate dates.

What is the best format for this? And where should it live?

Should it just be markdown? And if so, does it belong in one of the repos, or should we use HackMD or a wiki page?

Or a slide in Google docs?

Opinions/suggestions?

**Comment by hug-dev** [comment]
I think storing the roadmap in the community repository would be a good fit.

I am not sure of what format could be the best but something which is trackable in git offers great advantages I think. Right now I am thinking of markdown and also a duo of a PNG picture with its related draw.io XML file as we do in the book.

I have also seen some projects using a GitHub Project (the kanban board) to present their roadmap but there you lose the `git` trackability.

**Comment by ionut-arm** [comment]
> or a wiki page

I'd be in favor of using the [repo wiki page](https://github.com/parallaxsecond/parsec/wiki) - we've not used it at all, but I've seen it used for other projects, seems like a good alternative. 

EDIT: Then again, if Parsec will have a website of its own, we can just chuck it in there
